src/RoleObjects/SelectData.js

```diff
@@ -2,19 +2,19 @@ import GroupData from './GroupData';
 
 export default class SelectData extends GroupData {
   /**
-    * Sets the orientation
```

Comment:
Just happened to notice a comment spacing issue in this file, no functional changes here.